docs/src/main/asciidoc/tests-with-coverage.adoc

```diff
@@ -239,6 +247,20 @@ In addition to including the `quarkus-jacoco` extension in your pom you will nee
 WARNING: This config will only work if at least one `@QuarkusTest` is being run. If you are not using `@QuarkusTest` then
 you can simply use the Jacoco plugin in the standard manner with no additional config.
 
+If you are using Gradle, add this to your `build.gradle`:
+
+.Gradle
```

Comment:
I would remove the title here. 
And in the snippet, we also need to add the jacoco plugin in the `plugins` block